controllers/ebulletin/index.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+const path = require('path');
+const express = require('express');
+const Sentry = require('@sentry/node');
+const omit = require('lodash/omit');
+
+const { newContact, buildValidLocations } = require('./lib/contact-schema');
+
+const { noStore, csrfProtection } = require('../../common/cached');
+const { sanitiseRequestBody } = require('../../common/sanitise');
+const validateSchema = require('../../common/validate-schema');
+const newsletterService = require('../../common/ebulletin');
+const logger = require('../../common/logger').child({ service: 'ebulletin' });
+
+const router = express.Router();
+
+function subscribe(formData) {
+    const subscriptionData = {
```

Comment:
Rather than constructing this here, should this live in the newsletter service? Perhaps we just pass the `email` and `dataFields` at this point?